packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -414,20 +471,22 @@ export class MaterialPanel extends ConnectedLitElement {
   }
 
   onBaseColorChange() {
-    this.getMaterial().pbrMetallicRoughness.setBaseColorFactor(
-        this.selectedBaseColor);
+    const material = this.getMaterial(GetMaterialOptions.AllowInstancing);
```

Comment:
Maybe I'm just missing something, but where is the logic that changes `AllowInstancing`?

Reply:
The enum is just used to identify places in code where I want to get a material that could be instanced (if the material does not exist for a variant) no logic for setting/changing it just gets passed in, but it's moot it's gone now.